requirements/aarch64.txt

```diff
@@ -5,3 +5,4 @@ aiomysql==0.0.22
 aioboto3==9.0.0
 motor==2.5.1
 smbprotocol==1.9.0
+pymongo[srv]==3.13.0
```

Comment:
I'll create a follow-up issue here to investigate, whether we can use the same versions for `motor`.

Reply:
Hmmm how did it appear here? Should it be in a separate PR?

Reply:
Reason is in the PR description. Needed for testing it correctly

Reply:
I've tested connection to our cloud MongoDB instance without this change and it has been working all along. I will test again now, but what was the error that you've encountered during testing?

Reply:
`pymongo.errors.ConfigurationError: The "dnspython" module must be installed to use mongodb+srv:// URIs. To fix this error install pymongo with the srv extra:`. Maybe we're having issues in `make clean install` and something didn't install correctly? 🤔

Reply:
Maybe the version in `aarch64.txt` is too old to include this change: https://jira.mongodb.org/browse/MOTOR-284.

Reply:
Yep, running on M1.

Reply:
I see. I'd separate this change to a different PR and test it on all archs. Happy to help with x86_64 testing!

Reply:
Removed it, let's if the CI runs without an error with the `3.0.0` motor version